docs/intent_conversation_api.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Conversation API"
+sidebar_label: "Conversation API"
+---
+
+Intents can be recognized from text and fired using the [conversation integration](https://www.home-assistant.io/integrations/conversation/).
+
+An HTTP API endpoint is available at `/api/conversation/process`, which receives an [input sentence](#input-sentence) and produces an [intent response](#intent-response).
+
+## Input Sentence
+
+A sentence may be POST-ed to `/api/conversation/process` like:
+
+```json
+{
+  "text": "turn on the lights in the living room",
+  "language": "en"
+}
+```
+
+The following input fields are available:
+
+| Name              | Type   | Description                                                                     |
+|-------------------|--------|---------------------------------------------------------------------------------|
+| `text`            | string | Input sentence.                                                                 |
+| `language`        | string | Optional. Language of the input sentence (defaults to configured language).     |
+| `conversation_id` | string | Optional. Unique id to track conversation across multiple inputs and responses. |
```

Comment:
Should we mention that we will return conversation_id to users, and they shouldn't create their own ?

Reply:
We should, and I need to double check that I'm actually doing that...